models/reviews.js

```diff
@@ -27,8 +27,21 @@ exports.updateVote = (review_id, inc_votes) => {
     
 };
 
-exports.fetchReviews = () => {
-    return db.query(`
+exports.fetchReviews = (sort_by = 'created_at', order_by = 'DESC', category) => {
+    const queryValues = [];
+    const allowedCategories = [
```

Comment:
This is a good idea for now but later if we have the ability to add categories, this greenlist will constantly need updating. We could dynamically create the greenlist as we go or just skip this step altogether.